scripts/run_lint_python.sh

```diff
@@ -21,13 +21,16 @@ function display_result {
 flake8 --max-complexity 10 --count
 display_result $? 1 "Flake 8 code style check"
 
-pylint --reports=n --output-format=colorized --rcfile=.pylintrc -j 0 ./app ./tests ./scripts
+pylint --reports=n --output-format=colorized --rcfile=.pylintrc -j 0 **/*.py *.py
 # pylint bit encodes the exit code to allow you to figure out which category has failed.
 # https://docs.pylint.org/en/1.6.0/run.html#exit-codes
 # We want to fail on all errors so don't check for specific bits in the output; but if we did in future, see:
 # http://stackoverflow.com/questions/6626351/how-to-extract-bits-from-return-code-number-in-bash
 display_result $? 1 "Pylint linting check"
 
+isort --check **/*.py *.py
```

Comment:
Not sure why this isn't working. So I made change to import order to cause a failure, but when I run `make lint` it says there are no issues but if I run `pipenv run isort --check **/*.py *.py` manually then it gets picked up?.

Looks like isort isnt picking up the file at all, so `make format` isn't working either. The file I edited was `questionnaire.py`.

Reply:
when you say questionnaire.py, do you you mean `app/roots/questionnaire.py`?

Reply:
actually yeah I can see the issue - it's running it based off of the directory that run_test_lint is sat in

Reply:
Ok - I should have RTFM. There is a `--recursive` flag that works correctly